tools/fm-editor/Utils.h

```diff
@@ -0,0 +1,35 @@
+//
+// Created by simon on 20.02.23.
+//
+
+#ifndef VARA_FEATURE_UTILS_H
+#define VARA_FEATURE_UTILS_H
+
+#include <QMenu>
+template <class T>
+struct ActionBuilder {
+  ActionBuilder(QMenu *Menu, T *Receiver) : Menu(Menu), Receiver(Receiver) {}
+  template <class Func1, class... Func>
+  void addActions(std::pair<QString, Func1> Action,
+                  std::pair<QString, Func>... Actions) {
+    auto *A = new QAction(Action.first, Receiver);
+    QObject::connect(A, &QAction::triggered, Receiver, Action.second);
+    Menu->addAction(A);
+    addActions(Actions...);
+  }
+  void addActions() {}
```

Comment:
Why does this `addActions` method not take any inputs and does not have any impl?